test/srt/test_multi_image_openai_server.py

```diff
@@ -0,0 +1,95 @@
+import openai
```

Comment:
Now we also updated our unit test framework. Can you replace the old llava-1.5 with your new models?
https://github.com/sgl-project/sglang/blob/main/test/srt/test_vision_openai_server.py#L15-L31

This test is in CI so we will run it for every commit to make sure there is no regression.

Reply:
Yes, I modified it to `lmms-lab/llava-onevision-qwen2-0.5b-ov`.